ddtrace/propagation/_database_monitoring.py

```diff
@@ -25,23 +25,27 @@
 log = get_logger(__name__)
 
 
-def _default_sql_injector(dbm_comment, sql_statement):
+def default_sql_injector(dbm_comment, sql_statement):
     # type: (str, str) -> str
     try:
+        if isinstance(sql_statement, bytes):
+            try:
+                dbm_comment = dbm_comment.encode("utf-8", errors="strict")
+            except UnicodeDecodeError:
+                dbm_comment = ""
```

Comment:
```suggestion
                dbm_comment = b""
```

?

Reply:
if we have a decoding error, then we'd be trying to concatenate `str + bytes` which is no good